src/upgrade.rs

```diff
@@ -258,31 +258,31 @@ mod sealed {
         fn on_upgrade(self) -> OnUpgrade;
     }
 
-    impl CanUpgrade for http::Request<crate::Body> {
+    impl<T> CanUpgrade for http::Request<T> {
```

Comment:
This parameter is usually called `B` in most of Hyper.

Reply:
Thanks, updated.